tests/test_pix_initialization.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from staff import pxl
+
+
+@pytest.mark.parametrize("red, green, blue, expected_error", [
```

Comment:
You mixed two tests here: 3 for wrong pixel creation and 1 for successful creation, but here no tests for addition.
If you expect to write tests for creation then they should be separate - for successful and that raises exception because behavior is different.
If we test creation then I'd make valid tests for every component with boundary value and other is nominal. The same for wrong values, each component should be checked for all wrong values.